packages/twenty-server/src/engine/api/graphql/graphql-query-runner/resolvers/graphql-query-merge-many-resolver.service.ts

```diff
@@ -218,6 +243,238 @@ export class GraphqlQueryMergeManyResolverService extends GraphqlQueryBaseResolv
     return false;
   }
 
+  private mergeCompositeField(
```

Comment:
having to merge fields is not related to the fact they are composite but that their field or subfields is an "array" 
- ARRAY
- MULTI_SELECT
- PHONES
- LINKS

Address is composite but does not quality to be mergeable